docs/end-user/rollout.md

```diff
@@ -1,5 +1,417 @@
 ---
 title: Rollout
 ---
+This chapter will introduce how to use Rollout Trait to perform a rolling update on Workload.
 
-TBD
+## Background
+
+### ComponentRevision
+Each modification to a Trait will produce a kubernetes controllerRevision, the default role to generate a name of kubernetes controllerRevision is: `<Component name>-<revision number>`. You can also appoint controllerRevision name by setting `spec.components[x].externalRevision`.
+
+### Support workload type
+Rollout Trait support webservice,worker and cloneset Workload.
+
+When using webservice/worker as Workload type with Rollout Trait, Workload's name will be controllerRevision's name. And when Workload's type is cloneset-service, because of clonset support in-place update Workload's name will always be component's name.
```

Comment:
```suggestion
When using webservice/worker as Workload type with Rollout Trait, Workload's name will be controllerRevision's name. When the workload type is cloneset-service, because cloneset supports in-place update, the workload name will keep using the component name.
```